sdk/src/client/secret/mod.rs

```diff
@@ -66,6 +70,15 @@ use crate::{
 pub trait SecretManage: Send + Sync {
     type Error: std::error::Error + Send + Sync;
 
+    /// Generates public keys.
```

Comment:
Maybe add the same link for details on the coin type since it also applies for this method?

Reply:
I've done it quick and dirty to unblock myself because this is all going to change anyway. But I've added it.